app/helpers/application_helper.rb

```diff
@@ -2,11 +2,16 @@
 require 'json'
 
 module ApplicationHelper
-  def sanitize(user_input_text)
+  def sanitize_project_name(user_input_text)
```

Comment:
Could we move these functions into a new `ProjectsHelper`? Since they are related to projects only.